packages/features/profile/lib/getBranding.ts

```diff
@@ -0,0 +1,48 @@
+type EventTypeWithBranding = {
+  team?: {
+    name?: string;
+    brandColor?: string | null;
+    darkBrandColor?: string | null;
+    theme?: string | null;
+    parent?: {
+      brandColor?: string | null;
+      darkBrandColor?: string | null;
+      theme?: string | null;
+    } | null;
+  } | null;
+  profile?: {
+    organization?: {
+      brandColor?: string | null;
+      darkBrandColor?: string | null;
+      theme?: string | null;
+    } | null;
+  } | null;
+  users: Array<{
+    theme?: string | null;
+    brandColor?: string | null;
+    darkBrandColor?: string | null;
+  }>;
+};
+
+export function getBrandingForEventType(params: { eventType: EventTypeWithBranding }): {
+  theme: string | null;
+  brandColor: string | null;
+  darkBrandColor: string | null;
+} {
+  const { eventType } = params;
+
+  if (eventType.team) {
+    return {
+      theme: eventType.team.parent?.theme ?? eventType.team.theme ?? null,
+      brandColor: eventType.team.parent?.brandColor ?? eventType.team.brandColor ?? null,
+      darkBrandColor: eventType.team.parent?.darkBrandColor ?? eventType.team.darkBrandColor ?? null,
+    };
```

Comment:
I am not sure if this fallback is entirely correct.

My confusion:
What if _eventType.team.parent.theme_ is set but _eventType.team.parent?.brandColor_ isn't. Then we endup picking one thing from organization and other thing from team.


I think the right approach would be to check if parent exists and if it does use all the settings of it or use none.

```
const branding = team.parent ?? team

theme: branding.theme,
brandColor: branding.brandColor
darkBrandColor: branding.darkBrandColor
```

Similar is true for user branding